definitions/subworkflows/merge_svs.cwl

```diff
@@ -69,37 +50,19 @@ steps:
             cohort_name: cohort_name
         out:
             [merged_vcf]
-    add_population_frequency:
-        run: ../tools/add_sv_population_frequency.cwl
-        in:
-            vcf: merge_vcfs/merged_vcf
-            sv_db: sv_db
-            cohort_name: cohort_name
-        out:
-            [merged_annotated_vcf]
     annotate_variants:
-        run: ../tools/vep.cwl
+        run: ../tools/annotsv.cwl
         in:
-            vcf: add_population_frequency/merged_annotated_vcf
-            cache_dir: vep_cache_dir
-            ensembl_assembly: vep_ensembl_assembly
-            ensembl_version: vep_ensembl_version
-            ensembl_species: vep_ensembl_species
-            synonyms_file: synonyms_file
-            coding_only: coding_only
-            reference: reference
-            custom_gnomad_vcf: custom_gnomad_vcf
-            custom_clinvar_vcf: custom_clinvar_vcf
-            plugins: vep_plugins
-            everything:
-                default: false
-            pick:
-                default: "per_gene"
+            genome_build: genome_build
+            input_vcf: merge_sv_vcfs/merged_vcf
+            snps_vcf:
+                source: [snps_vcf]
+                valueFrom: ${ return [ self ]; }
         out:
-            [annotated_vcf, vep_summary]
-    sort_vcf:
-        run: ../tools/sort_vcf.cwl
+            [sv_variants_tsv]
+    merge_sv_bgzip:
```

Comment:
I might call this something like `bgzip_merged_sv_vcf`. The current name makes it sound like it is also performing a `merge` like the `merge_sv_vcfs` step above.

Reply:
I agree!